template/templateDAO.cabal

```diff
@@ -0,0 +1,35 @@
+cabal-version: 2.2
```

Comment:
Not sure why you added this (I guess it should be git-ignored too perhaps), but it is not necessary.

Reply:
I added it, because one of the checks mantioned, that I should update cabal files

Reply:
I think it's a reminder on your system from a previous version of the repo.

As you can see in `master` there is no longer a `./template` dir, because it has been moved in `./lorentz/template/`, so you can just delete this file (as well as the `template` line you added to the `.gitignore`

Reply:
Is is strange though, that I don't have this folder. locally and I also don't see it in my branch here, but the check is still failing

Reply:
Which one? The `./template` one? You should indeed not have that :thinking: 

As for the check it seems that it's working correctly as in [your latest CI run](https://buildkite.com/serokell/basedao/builds/1565#e67ef4b4-8364-4448-92a4-c40b6f4db382) the cabal step is passing.